test/test_restful.py

```diff
@@ -337,6 +338,9 @@ class TestRestful(unittest.TestCase):
     tpm_policy = {}
     metadata = {}
     allowlist = {}
+    excllist = {}
+    allowlist_bundle = {}
```

Comment:
`ima_bundle` ?

Reply:
I've made this and other naming changes in my latest push.